crates/subspace-core-primitives/src/lib.rs

```diff
@@ -19,6 +19,7 @@
 #![forbid(unsafe_code)]
 #![warn(rust_2018_idioms, missing_docs)]
 #![cfg_attr(feature = "std", warn(missing_debug_implementations))]
+#![feature(int_log)]
```

Comment:
The constants are moved to core primitives, `#![feature(int_log)]` in the subspace-runtime-primitives is now unnecessary.